patches/react-native+0.61.5.patch

```diff
@@ -19,31 +19,6 @@ index 7dffc17..548e7bb 100644
    },
    horizontallyInverted: {
      transform: [{scaleX: -1}],
-diff --git a/node_modules/react-native/Libraries/Text/Text/RCTTextShadowView.m b/node_modules/react-native/Libraries/Text/Text/RCTTextShadowView.m
-index 1543bca..a4502b7 100644
---- a/node_modules/react-native/Libraries/Text/Text/RCTTextShadowView.m
-+++ b/node_modules/react-native/Libraries/Text/Text/RCTTextShadowView.m
-@@ -177,6 +177,12 @@ - (void)postprocessAttributedText:(NSMutableAttributedString *)attributedText
- 
- - (NSAttributedString *)attributedTextWithMeasuredAttachmentsThatFitSize:(CGSize)size
- {
-+  static UIImage *placeholderImage;
-+  static dispatch_once_t onceToken;
-+  dispatch_once(&onceToken, ^{
-+    placeholderImage = [UIImage new];
-+  });
```

Comment:
I tried to go back in time to see the backstory of this patch to `<Text>`, and I was able to find the [relevant issue](https://github.com/facebook/react-native/commit/06599b3e594355a1d5062ede049ff3e333285516) in `react-native` but I was unable to see why this patch was introduced into Mattermost. The commit (425cfebf99ae9f61bfd2284873083abe7709f671) that brought in this patch is a generic commit that doesn't explain the reason for the patch, which would be helpful for debugging and audit.

Reply:
Cant really remember how it was affecting us, but this piece of code is now in the library